frontend/src/main/scala/bloop/testing/TestSuiteSelection.scala

```diff
@@ -0,0 +1,27 @@
+package bloop.testing
+
+import cats.data.NonEmptyList
+import bloop.bsp.ScalaTestSuiteSelection
+import TestSuiteSelection._
+
+/**
+ * https://github.com/sbt/junit-interface/pull/108#issuecomment-1002225661
+ */
+final case class TestSuiteSelection(private val suites: Map[SuiteName, NonEmptyList[TestName]]) {
+  def get(suiteName: SuiteName): Option[NonEmptyList[TestName]] = suites.get(suiteName)
+}
+
+object TestSuiteSelection {
```

Comment:
Aren't we duplicating ScalaTestSuiteSelection ?

Reply:
Yeah, I removed it